pkg/ipmi/ipmi.go

```diff
@@ -88,6 +94,34 @@ type systemInterfaceAddr struct {
 	lun      byte //nolint:unused
 }
 
+type StandardEvent struct {
+	Timestamp    uint32
+	GenID        uint16
+	EvMRev       uint8
+	SensorType   uint8
+	SensorNum    uint8
+	EventTypeDir uint8
+	EventData    [3]uint8
+}
+
+type OEMTsEvent struct {
+	Timestamp        uint32
```

Comment:
i'll leave it to pablo to decide if y'all want this, and i'm sorry for the many rounds of review...

but would it be useful to make this a `time.Time`? and have a marshal() method on `OEMTsEvent` that encodes the whole structure correctly turning time.Time into a uint32?

as a reviewer, i don't know if this uint32 is seconds from unix epoch, or nanoseconds from jan 1, 2000, or some other random thing. It'd be nice if I didn't have to know at all by just using `time.Time`.

Reply:
@pmazzini @hugelgupf My thought is the timestamp is written by BMC, so users don`t need to care about how to write timestamp.

Reply:
Oh, that seems confusing. Then why does the user have to specify it here? That's just how it is?

Reply:
When sending the `Add Sel Entry` command, the requested data needs 16 bytes, otherwise the command will fail, so I still defined the structure by IPMI spec.